[2024-08-08 08:05:08,370] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.extract_heroes 2024-08-08T08:00:05.160256+00:00 [queued]>
[2024-08-08 08:05:08,384] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.extract_heroes 2024-08-08T08:00:05.160256+00:00 [queued]>
[2024-08-08 08:05:08,384] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:05:08,385] {taskinstance.py:1069} INFO - Starting attempt 2 of 2
[2024-08-08 08:05:08,386] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:05:08,402] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): extract_heroes> on 2024-08-08T08:00:05.160256+00:00
[2024-08-08 08:05:08,406] {standard_task_runner.py:52} INFO - Started process 61 to run task
[2024-08-08 08:05:08,411] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dota_dag', 'extract_heroes', '2024-08-08T08:00:05.160256+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dota.py', '--cfg-path', '/tmp/tmpz8t2569v', '--error-file', '/tmp/tmp4xp6xjh6']
[2024-08-08 08:05:08,412] {standard_task_runner.py:77} INFO - Job 9: Subtask extract_heroes
[2024-08-08 08:05:08,463] {logging_mixin.py:104} INFO - Running <TaskInstance: dota_dag.extract_heroes 2024-08-08T08:00:05.160256+00:00 [running]> on host ec67c4f7e31a
[2024-08-08 08:05:08,505] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dota_dag
AIRFLOW_CTX_TASK_ID=extract_heroes
AIRFLOW_CTX_EXECUTION_DATE=2024-08-08T08:00:05.160256+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-08T08:00:05.160256+00:00
[2024-08-08 08:05:08,538] {python.py:118} INFO - Done. Returned value was: None
[2024-08-08 08:05:08,552] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=dota_dag, task_id=extract_heroes, execution_date=20240808T080005, start_date=20240808T080508, end_date=20240808T080508
[2024-08-08 08:05:08,586] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-08 08:05:08,622] {local_task_job.py:146} INFO - Task exited with return code 0
